### PR TITLE
Update-Webpack-Babel

### PR DESCRIPTION
**Pull request by mmarkelov**
Actually, I don't think that we can merge that right now or later. 
@gribnoysup what do you think about updating babel and webpack?
Or this is useless, cause **next** branch is in work?

### CODE REVIEW

File: tools/es2015-preset.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-
-module.exports = {
-  presets: [
-    [ buildPreset, {
-      loose: true,
-      modules: BABEL_ENV === 'es' ? false : 'commonjs'
-    } ]
-  ]
-}
+module.exports = function(api) {
```
Comment (gribnoysup) [inline tools/es2015-preset.js]: Can we rename this file to `env-preset` instead of `es2015-preset` to align it with the change of the base preset?

**Comment by gribnoysup** [comment]
Thanks for the update! I want to check it locally before merging, but unfortunately I don't have time yet. Sorry for the delay 😅

**Comment by mmarkelov** [comment]
@gribnoysup, that’s totally fine. It’s better to check out, of cause)

**Comment by gribnoysup** [comment]
Hey @mmarkelov! I was checking this branch locally and it doesn't seem to work for me, UMD build fails with webpack errors 🤔 

```sh
Building react-yandex-maps.js ...
Hash: 8c2deab75f457a3e8e1e
Version: webpack 4.20.2
Time: 877ms
Built at: 2018-10-01 16:35:26
 1 asset
Entrypoint main = main.js
 [0] external {"root":"React","commonjs2":"react","commonjs":"react","amd":"react"} 42 bytes {0} [built]
 [2] ./src/util/events.js 2.88 KiB {0} [built]
 [3] ./src/GeoObject.js 5.66 KiB {0} [built]
 [4] ./src/util/Fragment.js 300 bytes {0} [built]
 [5] ./src/Control.js 5.59 KiB {0} [built]
 [6] multi ./src/index.js umd/react-yandex-maps.js 40 bytes {0} [built]
 [7] ./src/index.js 1.67 KiB {0} [built]
 [8] ./src/YMaps.js 2.99 KiB {0} [built]
[11] ./src/util/api.js 2.71 KiB {0} [built]
[12] ./src/Map.js 6.44 KiB {0} [built]
[13] ./src/ObjectManager.js 6.52 KiB {0} [built]
[14] ./src/Clusterer.js 5.21 KiB {0} [built]
[15] ./src/Helpers.js 4.32 KiB {0} [built]
    + 3 hidden modules

ERROR in multi ./src/index.js umd/react-yandex-maps.js
Module not found: Error: Can't resolve 'umd/react-yandex-maps.js' in '/Users/s.petushkov/Projects/react-yandex-maps'
 @ multi ./src/index.js umd/react-yandex-maps.js main[1]
child_process.js:644
    throw err;
    ^

Error: Command failed: webpack src/index.js umd/react-yandex-maps.js
    at checkExecSyncError (child_process.js:601:13)
    at execSync (child_process.js:641:13)
    at exec (/Users/s.petushkov/Projects/react-yandex-maps/tools/build.js:13:3)
    at Object.<anonymous> (/Users/s.petushkov/Projects/react-yandex-maps/tools/build.js:32:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
```

Does the build command works for you?

**Comment by mmarkelov** [comment]
> Hey @mmarkelov! I was checking this branch locally and it doesn't seem to work for me, UMD build fails with webpack errors 🤔
> 
> ```shell
> Building react-yandex-maps.js ...
> Hash: 8c2deab75f457a3e8e1e
> Version: webpack 4.20.2
> Time: 877ms
> Built at: 2018-10-01 16:35:26
>  1 asset
> Entrypoint main = main.js
>  [0] external {"root":"React","commonjs2":"react","commonjs":"react","amd":"react"} 42 bytes {0} [built]
>  [2] ./src/util/events.js 2.88 KiB {0} [built]
>  [3] ./src/GeoObject.js 5.66 KiB {0} [built]
>  [4] ./src/util/Fragment.js 300 bytes {0} [built]
>  [5] ./src/Control.js 5.59 KiB {0} [built]
>  [6] multi ./src/index.js umd/react-yandex-maps.js 40 bytes {0} [built]
>  [7] ./src/index.js 1.67 KiB {0} [built]
>  [8] ./src/YMaps.js 2.99 KiB {0} [built]
> [11] ./src/util/api.js 2.71 KiB {0} [built]
> [12] ./src/Map.js 6.44 KiB {0} [built]
> [13] ./src/ObjectManager.js 6.52 KiB {0} [built]
> [14] ./src/Clusterer.js 5.21 KiB {0} [built]
> [15] ./src/Helpers.js 4.32 KiB {0} [built]
>     + 3 hidden modules
> 
> ERROR in multi ./src/index.js umd/react-yandex-maps.js
> Module not found: Error: Can't resolve 'umd/react-yandex-maps.js' in '/Users/s.petushkov/Projects/react-yandex-maps'
>  @ multi ./src/index.js umd/react-yandex-maps.js main[1]
> child_process.js:644
>     throw err;
>     ^
> 
> Error: Command failed: webpack src/index.js umd/react-yandex-maps.js
>     at checkExecSyncError (child_process.js:601:13)
>     at execSync (child_process.js:641:13)
>     at exec (/Users/s.petushkov/Projects/react-yandex-maps/tools/build.js:13:3)
>     at Object.<anonymous> (/Users/s.petushkov/Projects/react-yandex-maps/tools/build.js:32:1)
>     at Module._compile (module.js:652:30)
>     at Object.Module._extensions..js (module.js:663:10)
>     at Module.load (module.js:565:32)
>     at tryModuleLoad (module.js:505:12)
>     at Function.Module._load (module.js:497:3)
>     at Function.Module.runMain (module.js:693:10)
> ```
> 
> Does the build command works for you?

Ooops, that is my fault. It was just fine, cause I got **umd** directory. Fix it. Now it should be fine.

**Pull request merged**